src/pages/colab.js

```diff
@@ -1,48 +1,174 @@
 import React from 'react';
-import clsx from 'clsx';
-import Link from '@docusaurus/Link';
-import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import Layout from '@theme/Layout';
 import MarketplaceBanner from '../components/marketplace/MarketplaceBanner';
-
+import Link from '@docusaurus/Link';
 import styles from './exchange.module.css';
 import MarketplaceCards from '../components/marketplace/MarketplaceCards';
-import MarketplaceSidebar from '../components/marketplace/MarketplaceSidebar';
+import HomepageCard from '../components/homepage/HomepageCard';
 
 export default function Marketplace() {
-  const [filteredProduct, setFilteredProduct] = React.useState({"category": "colab", "tags": []});
-
-  const {siteConfig} = useDocusaurusContext();
-
-  const handleClick = (data) => {
-    var tempFilter = filteredProduct.tags.slice()
-
-    if (data.tag) {
-      const index = tempFilter.indexOf(data.tag);
-      if (index !== -1) {
-        tempFilter.splice(index, 1);
-      } else {
-        tempFilter.push(data.tag)
-      }
-    }
-
-    if (data.category) {
-      setFilteredProduct({"category": data.category, "tags": tempFilter})
-    } else {
-      setFilteredProduct({"category": filteredProduct.category, "tags": tempFilter})
-    }
-
-    
-  };
+  const [plugins] = React.useState({
+    category: 'colab-iiq-plugins',
+    tags: [],
+  });
+
+  const [rules] = React.useState({
+    category: 'colab-rules',
+    tags: ['identity-security-cloud'],
+  });
+
+  const [saas] = React.useState({
+    category: 'colab-saas-connectors',
+    tags: [],
+  });
+
+  const [transforms] = React.useState({
+    category: 'colab-transforms',
+    tags: [],
+  });
+
+  const [workflows] = React.useState({
+    category: 'colab',
+    tags: ['workflows'],
+  });
+
+  const [communityTools] = React.useState({
+    category: 'colab-community-tools',
+    tags: [],
+  });
+
   return (
     <Layout description="The SailPoint Developer Community has everything you need to build, extend, and automate scalable identity solutions.">
       <main>
-      <MarketplaceBanner />
-      <div className={styles.blogContainer}>
-          <div className={styles.blogSidbarContainer}><MarketplaceSidebar selectedCategory={filteredProduct.category} filterCallback={handleClick}/></div>
-          <div className={styles.blogCardContainer}><MarketplaceCards filterCallback={filteredProduct}/></div>
-      </div>
-      
+        <MarketplaceBanner />
+        <div>
+          <div className={styles.mainCard}>
+            <div className={styles.contentContainer}>
+              <div className={styles.gettingStartedText}>
+                <div className={styles.gettingStartedOne}>
+                  CoLab Marketplace
+                </div>
+                <div className={styles.gettingStartedTwo}>
+                  What is the CoLab?
+                </div>
+                <div className={styles.gettingStartedThree}>
+                  Jump into our{' '}
```

Comment:
@darrell-thobe-sp this text seems like it was left over and should be removed. If you read it on the built site in the PR, you'll see what I mean.